### PR TITLE
Copy certmanager related cmd to net-certmanager repo

### PR DESCRIPTION
**Pull request by ZhiminXiang**

### CODE REVIEW

File: cmd/controller/main.go [hunk 1]
```diff
@@ -0,0 +1,29 @@
+/*
+Copyright 2019 The Knative Authors
```
Comment (mattmoor) [inline cmd/controller/main.go:2]: Incorrect copyright year:
```suggestion
Copyright 2020 The Knative Authors
```

**Comment by ZhiminXiang** [comment]
/hold
This PR is blocked by https://github.com/knative/net-certmanager/pull/6

**Comment by ZhiminXiang** [comment]
Close this PR as it has been covered by #6.

**Pull request closed without merging**